Pinta.Core/Managers/IChromeManager.cs

```diff
@@ -30,7 +30,61 @@
 
 namespace Pinta.Core;
 
-public sealed class ChromeManager
+public interface IChromeManager
+{
+	PointI LastCanvasCursorPoint { get; set; }
+
+	Box? MainToolBar { get; }
+	Box ToolToolBar { get; }
+	Box ToolBox { get; }
+	Box StatusBar { get; }
+
+	IProgressDialog ProgressDialog { get; }
+	Gio.Menu AdjustmentsMenu { get; }
+	Gio.Menu EffectsMenu { get; }
+	Application Application { get; }
+	Window MainWindow { get; }
+	Widget ImageTabsNotebook { get; }
+
+	bool MainWindowBusy { get; set; }
+
+	void InitializeApplication (Gtk.Application application);
```

Comment:
I'd suggest only having the interface expose what's needed for effects for now. Many of these functions are only intended to be called during startup

Reply:
I've done just that 👍 I would re-think some of the structure, though